lib/avromatic.rb

```diff
@@ -43,8 +44,19 @@ def self.build_messaging!
 
   def self.prepare!
     nested_models.clear
-    on_initialize.call if on_initialize
+    preregister_models!
   end
+
+  def self.preregister_models=(value)
```

Comment:
Renamed to `models`. Based on discussion yesterday I'll probably also rename the method to `#eager_load_models`.